__tests__/hooks/useCoreFeed.test.js

```diff
@@ -67,37 +65,6 @@ describe('Main Feed should run correctly', () => {
     expect(result.current.searchHeight).toEqual(20);
   });
 
-  it('getDataFeeds should run correctly', async () => {
-    const {result} = renderHook(() => useCoreFeed(), {wrapper: Store});
-    axios.get.mockResolvedValue(responseMock);
-    act(() => {
-      const resp = result.current.getDataFeeds(10, false);
-      expect(resp).resolves.toEqual(responseMock);
-    });
-
-    const setMainFeed = jest.spyOn(mainFeedAction, 'setMainFeeds');
-    const saveToCache = jest.spyOn(cache, 'saveToCache');
-    act(() => {
-      result.current.handleDataFeeds(responseMock);
-    });
-    expect(setMainFeed).toHaveBeenCalled();
-    expect(saveToCache).toHaveBeenCalled();
-    act(() => {
-      result.current.handleDataFeeds(responseMock, 10);
-    });
-
-    expect(setMainFeed).toHaveBeenCalled();
-    expect(saveToCache).toHaveBeenCalled();
-  });
-
-  it('checkCacheFeed should run correctly', async () => {
-    const {result} = renderHook(() => useCoreFeed(), {wrapper: Store});
-    act(() => {
-      result.current.checkCacheFeed();
-    });
-    expect(getSpecificCache).toHaveBeenCalledTimes(1);
-  });
-
   it('onBlockCompleted should run correctly', async () => {
```

Comment:
mas @iwangsyah any reason why you deleted the test code?

Reply:
mas @fajarism , because before when I want to commit my work this test is error and blocked and as we disscus on group you said to comment this line but sonarcloud doesn't accept comment code, we must delete comment code
but this code long time age when this test still error